### ISSUE TITLE
Sony DPT-CP1 page turning issue

### ISSUE THREAD

**Issue by robertsjchen** [HappyZ/dpt-tools#158]
Hello, 

Not sure I'm the first guy encounter this issue: I managed root the device after I received from B&H(the original firmware version of the device is 1.4.xxx I remember). how ever, one issue very bothered me, every time I turn page(regardless using KOREADER or official DigitalApp app), I can hear very light sound like "zhi...."(very hard to make descriptions, you can image just like shutter sound during photo capturing with digital camera). no any sound heard if just click any button. do you think this could be device defect?

I tried to factory reset the device but failed, still didn't get any clue.

Thanks!

**Comment by abnerSK** [comment]
Hi robertsjchen,

I can confirm this light noise “issue” that you described also exists on my device. 

But dont worry, I'm sure it does **NOT** result from the root operation. If you put the unrooted device very close to your ear (I did this test yesterday when my DPT-CP1 remained unrooted), you hear the sound as well.

Another strange thing I found is: this "zhi....“ sound is more perceptible when turing page under **landscape mode** using KOREADER. Luckily most time I keep my device in portrait mode.
